vespalib/src/tests/stllike/hash_test.cpp

```diff
@@ -29,6 +29,14 @@ namespace {
     };
 }
 
+TEST("test that hashValue gives expected response")
```

Comment:
why remove that hashValue tests? are these the tests that ensure we are Java compliant?

Reply:
I think I only moved them. As far as I can see they are here.

Reply:
my bad. Too tired today